app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
+  belongs_to :post
+
+  def update_likes_counter(id)
+    post = Post.find(id)
+    post.likes_counter = post.likes.count
+    post.save
+  end
```

Comment:
- [ ] OPTIONAL
Similarly, it's a good practice to use one of the Rails Active Callbacks `after_save` to call the `update_like_counter` method every time a new like has been saved to the database, and when `after_save` is called mark the `update_like_counter` method as _private_ accordingly, which should be similar to:

```ruby
 # app/models/like.rb
  after_save :update_like_counter

  private

  def update_like_counter
    author.increment!(:likes_counter)
  end
```

Reply:
well resolved! ✔️